docs/ibc/light-clients/localhost/connection.md

```diff
@@ -0,0 +1,20 @@
+<!--
+order: 3
+-->
+
+# Localhost Connections
+
+The 09-localhost light client module integrates with core IBC through a single sentinel localhost connection.
```

Comment:
Does this mean that it's not possible to create extra connections other than this sentinel localhost connection? Not sure what was decided here in the end

Reply:
This is correct. It doesn't make sense to create multiple localhost connections since they would be identical, but the sentinel connection would be called "connection-localhost" while all others would be "connection-X" where X is a random number. It would make it more complicated on the engineering side to support this, and the sentinel localhost connection can support any type of channel to be created. There's no benefit to having multiple localhost connections from my point of view. I think the same logic holds that you don't have multiple "localhost" instances on your local machine. There's only one local host

Reply:
Mentioned in overview table